ui/ui-components/__tests__/smoketest.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from "@testing-library/react";
+
+function Welcome({ firstName }: { firstName: string }) {
+  return (
+    <>
+      <h1>Welcome, {firstName}</h1>
+      <p data-testid="p1">We are glad you are here, {firstName}.</p>
+    </>
+  );
+}
+
+test("has correct welcome text", async () => {
+  render(<Welcome firstName="John" />);
+  expect(screen.getByRole("heading")).toHaveTextContent("Welcome, John");
+  expect(screen.getByTestId("p1")).toHaveTextContent(
+    "We are glad you are here, John."
+  );
+});
```

Comment:
Here's a smoke test for the new `@testing-library/react` library that replaced enzyme

Reply:
One thing to note, you need to `render()` in each test, or `beforeEach` - it gets cleared at the end of each test.